lib/floki/flat_text.ex

```diff
@@ -6,33 +6,40 @@ defmodule Floki.FlatText do
 
   @type html_tree :: tuple | list
 
-  @spec get(html_tree) :: binary
+  @spec get(html_tree, binary) :: binary
 
   @doc """
   Get text nodes from first level of HTML nodes.
 
 
   ## Examples
 
-      iex> Floki.FlatText.get([{"a", [], ["The meaning of life is...", {"strong", [], ["something else"]}] }])
+      iex> Floki.FlatText.get([{"a", [], ["The meaning of life is...", {"strong", [], ["something else"]}] }], "")
```

Comment:
This example can remain the previous version, without the empty string (since it's already being showed in the line below).